src/components/SNTLmap.vue

```diff
@@ -2148,186 +2148,175 @@ export default {
-      "ak ak . us us us"
-      "ak ak . us us us"
-      "ak ak .us us us"
-      ". . . us us us";
+  #legendContainer{
+    grid-area: legend;
+    margin-bottom: 0px;
+    margin-left: 30px;
+    z-index: 1;
   }
   #grid-left{
-    width: 30vw; // careful editing this, it's sizing the maps to the same scale
-    //margin-left: 10vw;
+    grid-area: ak;
+    width: 190vw;
+    margin-right: 2.5vw; 
   }
-  #legendContainer {
-    margin-top: 0px;
-    margin-left: 20px;
+  #ak{
+    width: 110vw;// careful editing this, it's sizing the maps to be on the same scale
   }
-  #ak {
-   width: 60vw;// 2x the width of the containerthis needs to match with #usa to keep scaling constant
-   margin-left: 30px;
-   padding-top: 50px;
-  }
-  #grid-right {
-    width: 70vw;// careful editing this, it's sizing the maps to the same scale
-    margin-right: 2.5vw;
-    margin-left: 0px;
+  #grid-right{
+    grid-area: us;
+    width: 90vw;
+    margin-left: 70px;
   }
   #usa{
-    width: 100vw; // 2x the width of the container, get cut off (intentionally). needs to be mirror with alaska
-    margin-left: -25px;
+    width: 160vw;// careful editing this, it's sizing the maps to be on the same scale
+  }
+  #peak-container{
+    grid-area: peak;
+  }
+  #melt-container{
+    grid-area: melt;
+  }
+  #wy21-container{
+    grid-area: wy21;
   }
- 
-}
 
+  @media screen and (min-width: 1024px){
+    #melt-svg {
+      transform: translate(0, 0px);
+    }
+    #peak-svg {
+      transform: translate(0, -5px);
+    }
+    .map-grid{
+      grid-template-areas: 
+        ". legend legend us us us"
+        ". wy21 peak us us us"
+        ". wy21 peak us us us"
+        ". wy21 melt us us us"
+        ". wy21 melt us us us"
+        ". ak ak us us us"
+        ". ak ak us us us"
+        ". ak ak us us us"
+        ". ak ak us us us"
+        ". . . us us us";
+    }
+    #grid-left{
+      width: 30vw; // careful editing this, it's sizing the maps to the same scale
+    }
+    #legendContainer {
+      margin-top: 0px;
+    }
+    #ak {
+      width: 55vw;// 2x the width of the containerthis needs to match with #usa to keep scaling constant
+    }
+    #grid-right {
+      width: 70vw;// careful editing this, it's sizing the maps to the same scale
+      margin-right: 2.5vw;
+      margin-left: 0px;
+    }
+    #usa{
+      width: 80vw; // 2x the width of the container, get cut off (intentionally). needs to be mirror with alaska
+    }
+  }
 
+  @media screen and (min-height: 900px){
+    #melt-svg {
+      transform: translate(0, 5px);
+    }
+    #peak-svg {
+      transform: translate(0, 0px);
+    }
+    .map-grid{
+      grid-template-areas: 
+        ". legend legend us us us"
+        ". wy21 peak us us us"
+        ". wy21 peak us us us"
+        ". wy21 melt us us us"
+        ". wy21 melt us us us"
+        "ak ak . us us us"
+        "ak ak . us us us"
+        "ak ak . us us us"
+        "ak ak . us us us"
+        ". . . us us us";
```

Comment:
It'd be interesting to revisit this approach for Western U.S. focused projects coming up.